tuna/miopen/parse_miopen_args.py

```diff
@@ -37,6 +38,38 @@ def get_import_cfg_parser(
       'Import MIOpenDriver commands and MIOpen performance DB entries.',
       [TunaArgs.VERSION, TunaArgs.CONFIG_TYPE],
       with_yaml=with_yaml)
+
+  group = parser.add_mutually_exclusive_group()
+
+  group.add_argument(
+      '--add_framework',
+      dest='add_framework',
+      choices=[frm.value for frm in FrameworkEnum],
+      help='Populate framework table with new framework and version')
+  group.add_argument('--add_model',
+                     dest='add_model',
+                     choices=[model.value for model in ModelEnum],
+                     help='Populate table with new model and version')
+  group.add_argument('--print_models',
+                     dest='print_models',
+                     action='store_true',
+                     help='Print models from table')
+  group.add_argument('--add_benchmark',
+                     dest='add_benchmark',
+                     action='store_true',
+                     help='Insert new benchmark')
+
+  parser.add_argument('-b',
+                      '--batches',
+                      type=str,
+                      dest='batches',
+                      help='Batch sizes to iterate over in the given configs')
+  parser.add_argument('--batchsize',
+                      dest='batchsize',
+                      type=int,
+                      default=None,
+                      required=False,
```

Comment:
Wouldnt a model batchsize be required always since its part of the benchmarks table ?